modules/table/src/integrationTest/java/org/apache/ignite/internal/threading/ItKvRecordApiThreadingTest.java

```diff
@@ -207,6 +233,119 @@ private static RecordContext<Tuple> binaryRecordContext() {
         return new RecordContext<>(KEY_RECORD.toKeyTuple(), new Record(KEY, "one").toFullTuple(), new Record(KEY, "two").toFullTuple());
     }
 
+    @SuppressWarnings("rawtypes")
```

Comment:
Why do you need to use raw types in these tests? I was able to get rid of them by simply using the wildcard type everywhere